query/src/test/scala/hmda/query/model/institutions/InstitutionQueryGenerators.scala

```diff
@@ -7,20 +7,60 @@ object InstitutionQueryGenerators {
   implicit def institutionQueryGen: Gen[InstitutionQuery] = {
```

Comment:
I couldn't find anywhere in the project that this is used. Will there be a kind of test that uses query generators?